### ISSUE TITLE
Make Hexle more friendly to screen reader users

### ISSUE THREAD

**Issue by James-Livesey** [James-Livesey/hexle#5]
Cells should be annotated with WAI-ARIA attributes so that screen reader users can understand the state of each cell. Maybe also make it easier to check the outcome after entering it.
